Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM --platform=linux/amd64 rust:1.79-slim-bookworm
+
+LABEL maintainer="DecideAI"
+
+ENV NVM_DIR=/root/.nvm
+ENV NVM_VERSION=v0.39.7
+ENV NODE_VERSION=21.5.0
+ENV DFX_VERSION=0.19.0
```

Comment:
please make these consistent with the ci setup ( check yml files in .github/workflows )

Reply:
agree, its not consistent. we use v0.20.1

Reply:
It's overridden by the config in the docker compose, but yeah, fixing it